topics/tailwind/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: tailwind-css, tailwindcss, tailwind-scaffolding
+created_by: Adam Wathan, Jonathan Reinink, David Hemphill, Steve Schoger
+display_name: Tailwind CSS
+github_url: https://github.com/tailwindcss
+logo: tailwind.png
+related: framework, css, postcss, css-framework
+released: October 2017
+short_description: Tailwind is a Utility-First CSS framework.
+topic: tailwind
+url: https://tailwindcss.com/
+---
+A utility-first CSS framework for rapid UI development written in PostCSS and configured through JavaScript, which means it’s highly customizable by design. It doesn’t have a default theme and nor built-in UI components but this also means it has no opinion about how your site should look and doesn’t impose design decisions that you have to fight to undo.
```

Comment:
Can you rephrase the first part, maybe just by adding "Tailwind is" in front of "a utility-first CSS framework..."?

Maybe instead of "which means it’s highly customizable", "making it highly customizable"?

Maybe for the second sentence:

> Tailwind doesn’t have a default theme nor built-in UI components. This means it has no opinion about how your site should look and doesn’t impose design decisions that can be difficult to undo.